KomaMRICore/test/runtests.jl

```diff
@@ -501,6 +507,9 @@ end
 
 @testitem "Bloch_CPU_single_thread" tags=[:important, :core] begin
     using Suppressor, HDF5
+    include("../../KomaMRIIO/src/KomaMRIIO.jl")
+    using .KomaMRIIO
+
```

Comment:
Same here

Reply:
Addressed in [a7145d6](https://github.com/cncastillo/KomaMRI.jl/commit/a7145d642a633d7aa5d238007b66511b91241eda)